### PR TITLE
implement webview for opening resource links

### PR DESCRIPTION
**Pull request by hridayK**
- #11 
preview of change made:

webview_add

### CODE REVIEW

File: lib/Screens/OpenSource.dart [hunk 1]
```diff
@@ -124,12 +126,17 @@ class _OpenSourceState extends State<OpenSource> {
...
-                                            } else {
-                                              throw 'Could not launch $url';
-                                            }
+                                            Navigator.push(
+                                              context,
+                                              MaterialPageRoute(
+                                                builder: (context) =>
+                                                    WebViewWidget(
+                                                      title: showData[index]
+                                                      ['head'],
```
Comment (mdnazisharman2803) [inline lib/Screens/OpenSource.dart]: @hridayK  Add error handlling too if you can for like before, what if the page doesn't load

Comment (hridayK) [reply lib/Screens/OpenSource.dart]: image
added this

**Review by mdnazisharman2803 — APPROVED** [review]
Good work @hridayK .. Good to merge @Subhradeep10

**Comment by Subhradeep10** [comment]
@hridayK Resolve conflicts

**Comment by hridayK** [comment]
Resolved the merge conflicts

**Comment by mdnazisharman2803** [comment]
@hridayK  yeah it is resolved.. @Subhradeep10  will approve when he will see it.

**Pull request merged**